code/src/sixsq/nuvla/server/util/metadata.clj

```diff
@@ -130,35 +155,40 @@
   ([child-ns parent-ns spec]
    (generate-metadata child-ns parent-ns spec nil))
   ([child-ns parent-ns spec suffix]
+   (generate-metadata child-ns parent-ns spec suffix nil))
+  ([child-ns parent-ns spec suffix {:keys [scrud-operations] :as _options}]
```

Comment:
The `scrud-operations` option is currently not used, but idea is to provide a way to override the auto-discevery for cases where it does not work. I think for example where `defroutes` is used in resources defintions.